_docs-sources/pipelines/access-control/index.md

```diff
@@ -0,0 +1,23 @@
+# Access control
+
+Gruntwork Pipelines grants permissions by defining three [GitHub Teams](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams), which should map to three separate personas in your organization. Each team and its permissions are designed to grant the _least possible permissions_ to each individual (or machine user) in your organization for them to be able to perform changes to your infrastructure.
+
+### GitHub Team: infrastructure-administrators
+
+The `infrastructure-administrators` team is for engineers who likely work on the IaC codebase daily, and  have administrative AWS permissions. These users will also have permission to both read and write Pipelines configuration files, and the GitHub Actions Workflow files that are used to trigger Pipelines.
```

Comment:
When you say `pipelines configuration files` are you referring to the workflows in the infra-pipelines repository?

Reply:
I was making a bit of a leap here to include some future concept of configuration files that Pipelines admins will be able to edit. Feel free to updated as needed.

Reply:
Will do, thanks!